components/Players/PlayerManager.vue

```diff
@@ -195,6 +203,34 @@ export default Vue.extend({
         default:
           return this.getCurrentItem.Name;
       }
+    },
+    timeLeft(): number {
+      return parseInt((this.mediaDuration - this.getCurrentTime).toFixed());
+    },
+    mediaDuration(): number {
+      // In seconds
+      return this.ticksToMs(this.getCurrentItem.RunTimeTicks) / 1000;
+    },
+    showUpNext(): boolean {
+      if (
+        this.isMinimized ||
+        this.upNextUserHidden ||
+        this.getCurrentItem.Type !== 'Episode' ||
```

Comment:
What if I'm playing multiple movies (For example, if I'm watching Back to the Future or Lord of the Rings) ? It'd be good for this to be modular enough to support all media types, just like the audio player isn't inherently tied to music.

Reply:
Do we have queues for movies?

Reply:
Currently, everything has a queue.

In jf-web, you couldn't really make a queue with movies, I think (short of making a playlist), but with the floating player in this client, it's perfectly possible to launch playback, then queue other movies after it.

Reply:
Yeah ofc, I should've known better how I've played with it haha

Reply:
> Currently, everything has a queue.
Should we just show the dialog every time?